Manifests/ManagedPreferencesApplications/nl.root3.support.plist

```diff
@@ -244,6 +259,18 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_value_placeholder</key>
 			<string>ex. /Library/Application Support/Your Company/DarkModeLogo.png</string>
 		</dict>
+        	<dict>
+			<key>pfm_description</key>
+			<string>Path to a custom square image to be shown in alerts and the about window.</string>
+			<key>pfm_name</key>
+			<string>NotificationIcon</string>
+			<key>pfm_note</key>
+			<string>Some preference keys like the icon and status bar icon point to a file location. Due to the sandboxed characteristic of the app, not all file locations are allowed. We suggest putting the files in a folder within Application Support such as /Library/Application Support/Your Company/ where the app can read the contents. Other supported file locations can be found in Apple's documentation about App Sandbox: https://developer.apple.com/library/archive/documentation/Security/Conceptual/AppSandboxDesignGuide/AppSandboxInDepth/AppSandboxInDepth.html#/apple_ref/doc/uid/TP40011183-CH3-SW17</string>
+			<key>pfm_type</key>
```

Comment:
Can you please add a `pfm_title` for these?  This will end up displaying a more "user friendly" preference name rather than using the actual key name